hack/util

```diff
@@ -31,8 +31,10 @@ else
 fi
 
 buildxCmd() {
-  set -x
-  BUILDX_NO_DEFAULT_LOAD=true BUILDX_BUILDER="${builderName:-${BUILDX_BUILDER}}" $buildxBin "$@" $progressFlag
+  (
+    set -x
+    BUILDX_NO_DEFAULT_LOAD=true BUILDX_BUILDER="${builderName:-${BUILDX_BUILDER}}" $buildxBin "$@" $progressFlag
+  )
 }
 
 buildAttestFlags() {
```

Comment:
You could include `builder-id` attribute for provenance with the link to the current GH Action job.